doc/nrf/known_issues.rst

```diff
@@ -2010,6 +2010,13 @@ NCSDK-12483: Missing debug symbols
 
   **Workaround:** Add the following text in the application :file:`CMakeLists.txt` file: ``zephyr_link_libraries(-Wl,--undefined=my_missing_debug_symbol)``.
 
+.. rst-class:: v1-9-0
+
+NCSDK-14015: Execution halts during boot.
+  When rpmsg, used by openthread and BT, is enabled on nrf53 simultaneously with TF-M the firmware does not boot.
+
+  **Workaround:** Update to 1.9.1``.
```

Comment:
```suggestion

```
I would add this first in the 1.9-branch (I'm going to open a PR for known issues there). If we add it now to main, the workaround will not work.

Alternatively, you can add:
```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``XYZ``).
```
And then we update the workaround with the v1.9.1 option.

Reply:
I'm worried the SHA will change when the PR is rebased onto nrf so I decided to describe the fix instead.

I expect everyone that wants to apply the fix to either update to 1.9.1 or to use the main branch instead of 1.9.0.